src/apps/deskflow-gui/MainWindow.cpp

```diff
@@ -465,6 +494,25 @@ void MainWindow::resetCore()
   m_CoreProcess.restart();
 }
 
+void MainWindow::updateSize()
+{
+  if (ui->textLog->isVisible()) {
+    setMaximumHeight(16777215);
+    setMaximumWidth(16777215);
+#ifdef Q_OS_MAC
+    // On mac os the titlebar is part of the height so we need to adjust our Y coord to not "jump" up
+    setGeometry(x(), y() + 28, m_logRect.width(), m_logRect.height());
```

Comment:
For readability, other developers would find it helpful for the magic number to be a constant. This is optimized out by the compiler so it won't add a perf overhead.

```suggestion
    // On macOS, the titlebar is part of the height so we need to adjust our Y coord to not "jump" up
    const auto kTitlebarOffset = 28;
    
    setGeometry(x(), y() + kTitlebarOffset, m_logRect.width(), m_logRect.height());
```

Reply:
that is fine i was just trying to avoid making a var that was not gonna be used too often

Reply:
Yeah I know what you mean.